### PR TITLE
Docs: Correct spelling and Update documents

### PR DESCRIPTION
**Pull request by expitly**
1. Correct Korean Grammar.

2.  I have updated the revised contents in **Fixed some details related to the event loop (#1603)**.

### CODE REVIEW

File: locale/ko/docs/guides/event-loop-timers-and-nexttick.md [hunk 1]
```diff
@@ -158,10 +157,10 @@ any.
 
 * **timers**: 이 단계는 `setTimeout()`과 `setInterval()`로 스케줄링한
   콜백을 실행합니다.
-* **I/O callbacks**: 클로즈 콜백, 타이머로 스케줄링된 콜백,
-  `setImmediate()`를 제외한 거의 모든 콜백을 실행합니다.
+* **pending callbacks**: 다음 루프 반복으로 연기된 I/O 콜백들을 실행합니다.
 * **idle, prepare**: 내부용으로만 사용합니다.
-* **poll**: 새로운 I/O 이벤트를 가져옵니다. 적절한 시기에 node는 여기서 블록 합니다.
+* **poll**: 새로운 I/O 이벤트를 가져옵니다. 클로즈 콜백, 타이머로 스케줄링된 콜백,
+  `setImmediate()`를 제외한 거의 모든 콜백을 실행합니다. 적절한 시기에 node는 여기서 블록 합니다.
```
Comment (outsideris) [inline locale/ko/docs/guides/event-loop-timers-and-nexttick.md]:
```
새로운 I/O 이벤트를 가져옵니다. I/O와 연관된 콜백(클로즈 콜백, 타이머로 스케줄링된 콜백, `setImmediate()`를 제외한 거의 모든 콜백)을 실행합니다. 적절한 시기에 node는 여기서 블록 합니다.
``` 

가 되어야 할 것 같습니다.

File: locale/ko/docs/guides/event-loop-timers-and-nexttick.md [hunk 2]
```diff
@@ -142,12 +142,11 @@ _**NOTE:** 윈도우와 Unix/Linux 구현체간에 약간의 차이가 있지만
...
  and `setInterval()`.
-* **I/O callbacks**: executes almost all callbacks with the exception of
- close callbacks, the ones scheduled by timers, and `setImmediate()`.
+* **pending callbacks**: executes I/O callbacks deferred to the next loop iteration.
 * **idle, prepare**: only used internally.
-* **poll**: retrieve new I/O events; node will block here when appropriate.
+* **poll**: retrieve new I/O events; execute I/O related callbacks (almost all with the exception of close callbacks, the ones scheduled by timers, and `setImmediate()`); node will block here when appropriate.
 * **check**: `setImmediate()` callbacks are invoked here.
-* **close callbacks**: e.g. `socket.on('close', ...)`.
+* **close callbacks**: some close callbacks, e.g. `socket.on('close', ...)`.
```
Comment (yous) [inline locale/ko/docs/guides/event-loop-timers-and-nexttick.md:149]: This change isn't translated.

**Review by outsideris — APPROVED** [review]

**Review by yous — APPROVED** [review]

**Comment by fhemberger** [comment]
Thank you!

**Pull request merged**